easybuild/easyconfigs/c/cuQuantum/cuQuantum-22.05.0.41-CUDAcore-11.4.2.eb

```diff
@@ -0,0 +1,75 @@
+easyblock = 'Tarball'
+
+name = 'cuQuantum'
+version = '22.05.0.41'
+shortversion = '.'.join(version.split('.')[0:3])
+
+homepage = 'https://developer.nvidia.com/cuquantum-sdk'
+description = """NVIDIA cuQuantum SDK is a high-performance library for 
+quantum information science and beyond"""
+
+#cuStateVec requires CUDA 11.4 or above
+#python extension need scipy-stack
+#so gcccuda-2020.1.114
+toolchain = {'name': 'gcccuda', 'version': '2020.1.114'}
+toolchainopts = {'optarch': 'False'}
+
+# By downloading, you accept the cuQuantum Software License Agreement
+# (https://developer.nvidia.com/cuquantum-license-agreement)
+# accept_eula = True
+source_urls = ['https://developer.download.nvidia.com/compute/cuquantum/redist/cuquantum/linux-x86_64/cuquantum-linux-x86_64-%s-archive.tar.xz' %version]
+sources = ['cuquantum-linux-x86_64-%s-archive.tar.xz' %version]
+checksums = [
+    '3804ee39c96610ca1f1a644adde568344530f9377444b5309e20bc7d7e12ce57' #sources
+]
+
+dependencies = [
+    ('cuTENSOR', '1.5.0.3'),
+    ('SciPy-Stack', '2022a'),
+    ('CUDA', '11.4.2'),
+]
+
+builddependencies = {
+    ('python-build-bundle', '2022a'),
+    ('pytest', '7.0.1'),
+}
+
+multi_deps = {'Python' : ['3.8']}
+
+installopts = " && /cvmfs/soft.computecanada.ca/easybuild/bin/setrpaths.sh --path $EBROOTCUDA/lib --any_interpreter --add_path $EBROOTCUDA/lib"
```

Comment:
This is needed?

Reply:
corrected. It was a test for the python extension to find the cuda lib